src/Extender.php

```diff
@@ -133,21 +133,26 @@ public function processPackages(Event $event)
 
     protected function removeCacheFiles()
     {
+      try {
```

Comment:
Nice catch @guiajlopes 

Can we validate instead of try/catch, how about something like this:

```
if (is_dir(getcwd().'/console/cache/')) {
... code ...
}
```